packages/app-page-builder/src/editor/recoil/actions/cloneElement/action.ts

```diff
@@ -40,11 +43,14 @@ export const cloneElementAction: EventActionCallable<CloneElementActionArgsType>
     const parent = await state.getElementById(element.parent);
     const position = parent.elements.findIndex(el => el === element.id) + 1;
 
+    const newElementId = getNanoid();
```

Comment:
```suggestion
    const newElementId = randomIdGenerator();
```

Can we do that here?

Reply:
Oh, good catch man, I missed that one! 👍